packages/node-cli/src/commands/actionresult/ActionResultStatus.js

```diff
@@ -0,0 +1,8 @@
+/*
+** Copyright (c) 2019 Oracle and/or its affiliates.  All rights reserved.
+** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+module.exports = {
```

Comment:
This could be defined inside the `ActionResult` file and use multiple exports.

Reply:
Done